DefaultSettings.php

```diff
@@ -66,7 +66,7 @@
 	#
 	# @since 3.0
 	##
-	'smwgUpgradeKey' => 'smw:2018-09-01',
+	'smwgUpgradeKey' => 'smw:2019-01-19',
```

Comment:
@kghbln FYI Here is an example where we alter the upgrade key (as we are adding a new DB field) hereby forcing installations to run an appropriate update.